proto/viam/common/v1/common.proto

```diff
@@ -59,6 +61,10 @@ message Orientation {
 message PoseInFrame {
   string reference_frame = 1;
   Pose pose = 2;
+  // UUID of the geometry. If none supplied, will be an empty string.
+  bytes uuid = 3;
+  // Can hold information like color, opacity, points colors, collisionAllowed, etc...
+  google.protobuf.Struct metadata = 4;
```

Comment:
Same thing here.  I don't like adding fields to things that should just be geometry related.  Especially something like an unstructured metadata struct.  I would like to understand why these are necessary

Reply:
Per the email:

> Some things that people draw are not geometries (like poses represented as arrows)


We will want a way to provide rendering data like color and opacity for those cases.